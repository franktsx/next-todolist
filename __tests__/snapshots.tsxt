import { render } from '@testing-library/react'
import Index from '../pages/index'


it('renders homepage unchanged', () => {
  const { container } = render(<Index todos={[]} />)
  console.log(container)
  expect(container).toMatchSnapshot()
})