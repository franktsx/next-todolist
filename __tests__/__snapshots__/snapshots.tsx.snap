// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <h1>
      My Todo List
    </h1>
    <h2>
      Click On Todo to see it individually and editing
    </h2>
    <button>
      Create a New Todo
    </button>
    <div
      class="header"
    >
      <input
        placeholder="Search by item name..."
        type="text"
      />
    </div>
  </div>
</div>
`;
